### DISCUSSION TITLE
Request to add "Explore" section.

### DISCUSSION THREAD

**Discussion by ghost** [tuskyapp/Tusky#2907] [Ideas]
[Feature Request] In Tusky, we have tabs for "Local" and "Federated" posts but there is no tab available for "Explore" section (which contains popular or selected posts of different servers).
Please consider adding it. Thank You. 

![IMG_20221122_184428](https://user-images.githubusercontent.com/118822445/203325334-405bcbc0-fd7e-4f3f-8d92-d588edb58d59.jpg)
![IMG_20221122_182510](https://user-images.githubusercontent.com/118822445/203325348-edb2d481-3493-4aad-9cd7-02330f6cf1c9.jpg)
![IMG_20221122_182431](https://user-images.githubusercontent.com/118822445/203325359-ae15b98f-4543-43df-8fc6-bfd62d6ebe09.jpg)

**Comment by tinsukE** [comment]
The Explore TL is a great source for popular toots when you're done with your own TL, and a decent channel for discovering people to follow.

**Comment by nikclayton** [comment]
Mostly coming in https://github.com/tuskyapp/Tusky/pull/3910

Note to self: "explore" in the web UI also includes https://docs.joinmastodon.org/methods/suggestions/, filtering the results by calling https://docs.joinmastodon.org/methods/accounts/#relationships and checking to see that they're not already being followed.
